standards/qa-process.md

```diff
@@ -38,14 +38,16 @@
     - Task description
     - Due date (or estimate of testing time required)
     - Names of PM and lead dev
-    - Is a Report of Issues required?
 ```
 2. QA leads receives the project and assigns it to the QA team / PM on monday.com. Deadline is set up, a timeline is scheduled, and calendar invites are sent.
 
 ## Testing Kickoff
 1. Testing kickoff meeting occurs. PM or Dev will introduce the QA team to the Project-Under-Test.
 2. QA team meets to review the application and its purpose in order to better understand the most important functions of the website before diving in.
-3. PM / Dev defines the priorities for testing and defines the “Definition of Done”.
+3. PM / Dev defines the priorities for testing and defines the “Definition of Done”. 
+    - Are there any specific devices we should be looking at?
+    - What types of testing will we be doing for this project?
+    - Are there any forms of [compliance](https://www.google.com/url?q=https://docs.google.com/document/d/16_9HWZa2lmvAUeWNv0WD68oRlruZHlG3xGVGqg3EScw/edit?usp%3Dsharing&sa=D&source=editors&ust=1620757751653000&usg=AOvVaw1-rFvNAvKO15isaEe1P2oL) that need to be followed?
```

Comment:
Since this is a public repo, is this document protected by our organization? Can others outside the org see this doc? I just want to confirm this doc is protected.